hack/.golint_failures

```diff
@@ -59,6 +59,7 @@ pkg/controller/disruption
 pkg/controller/endpoint
 pkg/controller/endpoint/config/v1alpha1
 pkg/controller/endpointslice/config/v1alpha1
+pkg/controller/endpointslicemirroring/config/v1alpha1
```

Comment:
Can we just fix the lint errors?

Reply:
Unfortunately I think this is a problem with the required conversion functions like `Convert_v1alpha1_EndpointSliceMirroringControllerConfiguration_To_config_EndpointSliceMirroringControllerConfiguration`. As far as I can tell, every controller has an entry for this config in `.golint_failures`.